src/Provider/GoogleMaps/Model/SubLocalityLevel.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Geocoder package.
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @license    MIT License
+ */
+
+namespace Geocoder\Provider\GoogleMaps\Model;
+
+/**
+ * Class SubLocalityLevel is used only for GoogleMap provider and contains functions for working with concrete subLocalityLevel
+ */
+final class SubLocalityLevel
```

Comment:
This class is exactly the same as admin level. Maybe we do not need to create a new class